debezium-server/debezium-server-redis/src/test/java/io/debezium/server/redis/RedisDatabaseHistoryTestProfile.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.server.redis;
+
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import io.debezium.testing.testcontainers.MySqlTestResourceLifecycleManager;
+import io.debezium.util.Testing;
+import io.quarkus.test.junit.QuarkusTestProfile;
+
+public class RedisDatabaseHistoryTestProfile implements QuarkusTestProfile {
+    public static final String OFFSETS_FILE = "file-connector-offsets.txt";
+    public static final Path OFFSET_STORE_PATH = Testing.Files.createTestingPath(OFFSETS_FILE).toAbsolutePath();
+    public static final String OFFSET_STORAGE_FILE_FILENAME_CONFIG = "offset.storage.file.filename";
```

Comment:
Why do we need those file-related attributes here?

Reply:
I assumed the user can mix and match, so did not want to assume they use the RedisOffset adapter. So opted to keep the same File offset adapter to isolate the tests only to the DB History.

Reply:
Ah, ok, makes sense. In general I'd assume a user doesn't mix different implementations (in particular once we enable the usage of (Redis) transactions for doing these changes to offsets/events/history atomically), but it's fine for this test.

Reply:
I think the ideal would be to just specify sink=redis and then get the full support. But I thought about it later and can see reasoning for using Redis to store offsets and database history while using a different stream to achieve a stateless deployment of Debezium server without depending on local file system